swim.go

```diff
@@ -66,6 +68,8 @@ func (n *Node) runTick() {
 			n.send(n.tick())
 		case <-pingTimer.C:
 			n.send(n.timeout())
+		case <-n.doneTick:
```

Comment:
Nit: I would call this `stopTick`, since I personally usually use "done" to describe something that has already finished and "stop" when I need to tell something it should finish.

Reply:
I know I suggested `doneTick` in the first place, I just hadn't thought about it carefully sorry lol

Reply:
I like that.